src/js/post.js

```diff
@@ -112,7 +112,6 @@ const prepareProgressCircle = () => {
 $(document).ready(() => {
   $aosWrapper = $('.js-aos-wrapper')
   const $scrollButton = $('.js-scrolltop')
-  const $loadComments = $('.js-load-comments')
   const $commentsIframe = $('.js-comments-iframe')
```

Comment:
Would you mind to remove this variable as well? :)

Reply:
Haha thanks for the alert